deeplinkdispatch-processor/build.gradle

```diff
@@ -26,7 +25,6 @@ dependencies {
   testImplementation deps.compileTestingKsp
   testImplementation deps.mockk
   testImplementation deps.kotlinReflect
-  testImplementation files("${System.properties['java.home']}/../lib/tools.jar")
```

Comment:
How did you work around this? This line was bugging me for so long but I had no idea how to loose that dependency.